community/contributing/updating_the_class_reference.rst

```diff
@@ -1,24 +1,13 @@
-.. _doc_updating_the_class_reference_with_git:
+.. _doc_updating_the_class_reference:
 
-Contributing to the class reference with Git
+Contributing to the class reference
 ============================================
```

Comment:
```suggestion
===================================
```

The line below the header name should have the same number of `=` or `-` as the header name has characters.